features/scripts/build-plain-spring-app.sh

```diff
@@ -2,5 +2,10 @@
 
 # Start Tomcat then copy a WAR file across to serve it
 catalina.sh start
-./gradlew -p features/fixtures/mazerunnerplainspring war
-cp features/fixtures/mazerunnerplainspring/build/libs/mazerunnerplainspring.war $CATALINA_HOME/webapps/ROOT.war
+if [[ "${JAVA_VERSION}" == "8"* ]]; then
```

Comment:
Added to switch maze-runner projects mazerunnerplainspring/mazerunnerplainspring6